hawc/apps/assessment/exports.py

```diff
@@ -39,6 +39,7 @@ def get_annotation_map(self, query_prefix: str) -> dict:
             "uncertainty_display": sql_display(
                 query_prefix + "uncertainty", constants.UncertaintyChoices
             ),
+            "studies__short_citation": str_m2m(query_prefix + "studies__short_citation"),
```

Comment:
is this enough study information for this export? It's tricky I think to add the identifiers like pmid because it's multiple m2m relations. I also tried to add the hawc ID but str_m2m doesn't work on the ID field because it's an integer